### ISSUE TITLE
Hide Card ID for binary content?

### ISSUE THREAD

**Issue by leoheitmannruiz** [CatimaLoyalty/Android#1824]
Hey :)

First off:
Thanks for all the great work!!
I used to just keep images of barcodes in my gallery, but Catima is soo much nicer.
Especially that the screen brightness is automatically set to max when viewing barcodes!

---

I've got an aztec barcode, whose content is binary gibberish.
I wonder if it makes sense not to show the content/card id below the barcode in such cases.
It seems desirable to me, as (in my case) half of the characters are weird and the other half aren't displayed correctly.
Not very pretty.
I might be missing something, but generally I don't see a point in showing binary data.

What do you think?

**Comment by TheLastProject** [comment]
Yeah, it makes sense to hide binary. Just wondering what to do instead. Hide it completely? Show some text like "View barcode" you can still tap like with non-binary values?

And of course, detecting binary will be its own challenge.

**Comment by TheLastProject** [comment]
After working on #2716 I started to realize the line of "binary" is a bit blurry. For example the German word "Käseschnitte" is already UTF-8. So I'm not sure how to identify between "gibberish binary" and "languages that use special characters".

So that's a problem that would need solving first :)
